bin/medic.js

```diff
@@ -55,6 +181,210 @@ function exclusiveLs(lsPath, excludes) {
+    medicLog("Setting entry point to " + entryPoint + " in config.xml");
+
+    if (configContent.match(/<content\s*src=".*"\s*\/>/gi)) {
+        configContent = configContent.replace(
+            /<content\s*src=".*"\s*\/>/gi,
+            "<content src=\"" + entryPoint + "\" />"
+        );
+
+    } else {
+
+        // add entry point to config
+        configContent = configContent.split("</widget>").join("") +
+            "    <content src=\"" + entryPoint + "\" />\n</widget>";
+    }
+
+    // write the changes
+    fs.writeFileSync(configFile, configContent, DEFAULT_ENCODING);
+}
+
+function changeLoadTimeout(appPath, timeout) {
+
+    medicLog("Increasing url loading timeout for android to " + timeout);
+
+    var timeoutRegex           = /<preference\s*name\s *= \s*"?loadUrlTimeoutValue"?.*?((\/>)|(>.*?<\/\s*preference>))/i;
+    var timeoutTag             = "<preference name=\"loadUrlTimeoutValue\" value=\"" + timeout + "\" />";
+    var timeoutTagWithPlatform = "    <platform name=\"android\">\n        <preference name=\"loadUrlTimeoutValue\" value=\"120000\" />\n    </platform>\n";
+    var platformRegex          = /<platform\s*name\s *= \s*"android"\s*>/i;
+    var widgetRegex            = /<\/s*widget\s*>/i;
+
+    var configFile    = getConfigPath(appPath);
+    var configContent = fs.readFileSync(configFile, DEFAULT_ENCODING);
+
+    if (timeoutRegex.test(configContent)) {
+        configContent = configContent.replace(timeoutRegex, timeoutTag);
+        medicLog("Found \"loadUrlTimeoutValue\" preference, replacing with desired value");
+    } else if (platformRegex.test(configContent)) {
+        var oldPlatformTag = platformRegex.exec(configContent)[0];
+        configContent = configContent.replace(platformRegex, oldPlatformTag + "\n        " + timeoutTag);
+        medicLog("Found platform tag, appending \"loadUrlTimeoutValue\" preference");
+    } else if (widgetRegex.test(configContent)) {
+        var oldWidgetTag = widgetRegex.exec(configContent)[0];
+        configContent = configContent.replace(widgetRegex, timeoutTagWithPlatform + oldWidgetTag);
+        medicLog("Did not find platform tag, adding preference with platform tag");
+    } else {
+        medicLog("Warning: could not modify config.xml for android: no <widget> tag found!");
+    }
+
+    // write the changes
+    fs.writeFileSync(configFile, configContent, DEFAULT_ENCODING);
+}
+
+function setWindowsTargetStoreVersion(appPath, version) {
+
+    medicLog('setting target store version to ' + version);
+
+    var configFile    = getConfigPath(appPath);
+    var configContent = fs.readFileSync(configFile, DEFAULT_ENCODING);
+
+    var versionPreference = '    <preference name="windows-target-version" value="' + version + '" />';
+    configContent = configContent.replace('</widget>', versionPreference + '\r\n</widget>');
+
+    fs.writeFileSync(configFile, configContent, "utf8");
+}
+
+function androidSpecifics(argv) {
```

Comment:
Your "(platform)Specifics" functions don't really indicate what it is they do by their names.